parser/worker.go

```diff
@@ -121,6 +121,15 @@ func (w *Worker) Process(height uint64) error {
 
 	if err == nil {
 		log.Infow("processed block", "height", height)
+
+		totalBlocks := w.db.GetTotalBlocks(context.TODO())
+		log.DBBlockCount.Set(float64(totalBlocks))
```

Comment:
As `blocks` table is not used by block sycner, metrics here in Process method is not general. It's better to add a **metrics** interface as a property in Worker struct for different metrics.